### DISCUSSION TITLE
Navigation bar setup in tabs in header

### DISCUSSION THREAD

**Discussion by Shivansh-Mittal** [squidfunk/mkdocs-material#3979] [Q&A] [answered]
How can i setup my header navigation tabs like you used in mkdocs material webapp ?

![image](https://user-images.githubusercontent.com/81565355/171846849-559f0f50-1ba1-4c5a-9a3e-2a8e2ab32a09.png)

I need to setup for documentation for three different users (admin, teachers and students) and a home page.

**Comment by squidfunk** [comment, marked as answer]
The feature is called `navigation.tabs`:
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs

I really recommend reading the documentation – it explains all options and includes screenshots showcasing different features.
